src/instrumentations/pg/PgInstrumentation.test.js

```diff
@@ -1,12 +1,27 @@
 import LumigoPgInstrumentation from './PgInstrumentation';
+import child_process from 'child_process';
 
 describe('LumigoPgInstrumentation', () => {
+  const oldEnv = Object.assign({}, process.env);
+  beforeEach(() => {
+    process.env = { ...oldEnv };
+  });
+
   afterEach(() => {
     jest.clearAllMocks();
+    process.env = { ...oldEnv };
   });
 
   let lumigoPgInstrumentation = new LumigoPgInstrumentation();
 
+  test('disable pg instrumentation', () => {
```

Comment:
I'd go further and replace this with an actual instrumentation test, that sets the env-var to `false` and makes sure no spans were produced - see an example of [something similar I've done with the aws-sdk V3 ](https://github.com/lumigo-io/opentelemetry-js-distro/blob/main/test/instrumentations/%40aws-sdk/client-sqs/aws-sdk-v3.test.ts#L262).